sdk/bundles/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@piiano/bundles",
+  "version": "0.0.1",
+  "description": "Piiano Vault type definitions for bundles",
+  "types": "dist/index.d.ts",
+  "author": "Ori Shalom <ori.shalom@piiano.com>",
+  "files": [
+    "dist"
+  ],
+  "keywords": [
+    "Piiano",
```

Comment:
is that case sensitive?

Reply:
Looks like both
Capitalized https://www.npmjs.com/search?q=keywords%3APiiano
and
Lowercase https://www.npmjs.com/search?q=keywords%3Apiiano
yields the same results.